src/components/common/AddressInput/index.test.tsx

```diff
@@ -82,13 +82,6 @@ describe('AddressInput tests', () => {
   it('should validate the address on input', async () => {
     const { input, utils } = setup('')
 
-    act(() => {
-      fireEvent.change(input, { target: { value: `xyz:${TEST_ADDRESS_A}` } })
-      jest.advanceTimersByTime(1000)
-    })
-
-    await waitFor(() => expect(utils.getByLabelText('Invalid chain prefix "xyz"', { exact: false })).toBeDefined())
```

Comment:
Allowing invalid prefixes somewhat negates ERC-3770. Can we validate against the prefixes currently in the store?

Reply:
It still checks that the prefix matches the current chain, that's more than enough. No need to check it against all possible chains.